Client/src/Utils/Theme/constants.js

```diff
@@ -109,12 +110,12 @@ const semanticColors = {
 	},
 	error: {
 		main: {
-			light: paletteColors.red300,
-			dark: paletteColors.red300,
+			light: paletteColors.red400,
+			dark: paletteColors.red400,
 		},
 		contrastText: {
-			light: paletteColors.gray50,
-			dark: paletteColors.gray50,
+			light: paletteColors.red300,
+			dark: paletteColors.red300,
```

Comment:
I'm not sure about this change, gray to red is quite a change :laughing:   Please wait for @marcelluscaio 's review on this one as he is more familiar with the colors than I am.

Reply:
Got it. Will wait for the feedback.

Reply:
I don't think red 300 is a good color for the contrast text. I would keep colors as they are, and change the component that is calling the colors. I suggest you revert the changes made to the theme, and try to add another color to the component (that probably is calling error.contrastText). If you cannot work with the colors we currently have, the we can think of a way of adding the new shade of red. What do you think?

Reply:
Sounds good! I think adding a new shade of red would be the way to go. We can revert back the `error.contrastText` to use `gray50`. 

For the red colors, I inserted the `#EBA8A9` to be new value of the `red300` and created `red400` to be `#D32F2F` because it's a darker shade of red than the one I inserted.

Then we can just directly use new value of the `red300` into the chart.

What do you think?

Reply:
I would try to solve with the current colors before adding a new one. Let's add the #EBA8A9 only if we cannot solve with our current ones

Reply:
Got it. I think @gorkem-bwl will be able to help us with this.

These are the shades of color that we have right now:
#F9ECED
#FBD1D1
#F04438
#D32F2F
#542426
#301A1F

Are there any of the colors that you think we can use for the chart? Or should we stick with the #EBA8A9 as the color of the chart?

Reply:
I think you can go with
#FBD1D1
over
#F04438

We need to check light and dark modes. main prblem here was the use of contrastText, that is gray

Reply:
Here is how the chart looks right now using `#FBD1D1`:

![image](https://github.com/user-attachments/assets/5000891f-335e-4ef9-bc37-711f4c78f255)

I used the `error.light` color palette for this.

What do you think?

Reply:
I like how it looks, we would need to see dark mode as well. You haven't committed that yet right?

Reply:
Yep. I haven't committed the changes yet.

Here is how it looks in dark mode:
![image](https://github.com/user-attachments/assets/de667f7c-b4a9-4f5f-a135-2d4867d908e6)

Reply:
> Yep. I haven't committed the changes yet.
> 
> Here is how it looks in dark mode: ![image](https://private-user-images.githubusercontent.com/57168579/395248961-de667f7c-b4a9-4f5f-a135-2d4867d908e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMjE3NDksIm5iZiI6MTczNDAyMTQ0OSwicGF0aCI6Ii81NzE2ODU3OS8zOTUyNDg5NjEtZGU2NjdmN2MtYjRhOS00ZjVmLWExMzUtMmQ0ODY3ZDkwOGU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEyVDE2MzcyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJiZmI4MDQ5NDlmMmY3MDcwNTk0MGQ1YmEzMWZhYTAwMTE4NzJkMWIyYzk3N2Y0OTUxYzAxNjYzZGQ1NjA4YWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Vl1whXW_UQxayqi4Eej7gUDZFneT0PG_ddQ7ngbscNs)

It's using the `red700`: `#542426` as the main color and `red800`: `#301A1F` as the bg color.

Reply:
I think you can commit that. It solves the original issue, and we can open a new one if we need to adjust colors - adjusting colors impacts the whole project and goes beyond the scope of this issue

Reply:
Sound good! Thank you for reviewing this PR @marcelluscaio. I really appreciate it.

Reply:
I am the one to thank you for contributing to the project =)